docs/changelog/101915.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 101915
+summary: Add inference counts by model to the machine learning usage stats
+area: Machine Learning
+type: enhancement
+issues: []
```

Comment:
What's this used for? (It's auto-generated.)

Should I add the Github issue here?

Reply:
This is an automatically generated change log that is used when building the release notes. The bot will automatically create these for PR's with the `>enhancement` or `>bug` label. 

`>tests` and `>non-issue` labels will not create the entry.